app/experimenter/experiments/migrations/0202_auto_20220111_1630.py

```diff
@@ -0,0 +1,219 @@
+                    ("33.!", "Firefox 33"),
+                    ("34.!", "Firefox 34"),
+                    ("35.!", "Firefox 35"),
+                    ("36.!", "Firefox 36"),
+                    ("37.!", "Firefox 37"),
+                    ("38.!", "Firefox 38"),
+                    ("39.!", "Firefox 39"),
+                    ("40.!", "Firefox 40"),
+                    ("41.!", "Firefox 41"),
+                    ("42.!", "Firefox 42"),
+                    ("43.!", "Firefox 43"),
+                    ("44.!", "Firefox 44"),
+                    ("45.!", "Firefox 45"),
+                    ("46.!", "Firefox 46"),
+                    ("47.!", "Firefox 47"),
+                    ("48.!", "Firefox 48"),
+                    ("49.!", "Firefox 49"),
+                    ("50.!", "Firefox 50"),
+                    ("51.!", "Firefox 51"),
+                    ("52.!", "Firefox 52"),
+                    ("53.!", "Firefox 53"),
+                    ("54.!", "Firefox 54"),
+                    ("55.!", "Firefox 55"),
+                    ("56.!", "Firefox 56"),
+                    ("57.!", "Firefox 57"),
+                    ("58.!", "Firefox 58"),
+                    ("59.!", "Firefox 59"),
+                    ("60.!", "Firefox 60"),
+                    ("61.!", "Firefox 61"),
+                    ("62.!", "Firefox 62"),
+                    ("63.!", "Firefox 63"),
+                    ("64.!", "Firefox 64"),
+                    ("65.!", "Firefox 65"),
+                    ("66.!", "Firefox 66"),
+                    ("67.!", "Firefox 67"),
+                    ("68.!", "Firefox 68"),
+                    ("69.!", "Firefox 69"),
+                    ("70.!", "Firefox 70"),
+                    ("71.!", "Firefox 71"),
+                    ("72.!", "Firefox 72"),
+                    ("73.!", "Firefox 73"),
+                    ("74.!", "Firefox 74"),
+                    ("75.!", "Firefox 75"),
+                    ("76.!", "Firefox 76"),
+                    ("77.!", "Firefox 77"),
+                    ("78.!", "Firefox 78"),
+                    ("79.!", "Firefox 79"),
+                    ("80.!", "Firefox 80"),
+                    ("81.!", "Firefox 81"),
+                    ("82.!", "Firefox 82"),
+                    ("83.!", "Firefox 83"),
+                    ("84.!", "Firefox 84"),
+                    ("85.!", "Firefox 85"),
+                    ("86.!", "Firefox 86"),
+                    ("87.!", "Firefox 87"),
+                    ("88.!", "Firefox 88"),
+                    ("89.!", "Firefox 89"),
+                    ("90.!", "Firefox 90"),
+                    ("91.!", "Firefox 91"),
+                    ("92.!", "Firefox 92"),
+                    ("92.0.1", "Firefox 9201"),
+                    ("93.!", "Firefox 93"),
+                    ("94.!", "Firefox 94"),
+                    ("95.!", "Firefox 95"),
+                    ("96.!", "Firefox 96"),
+                    ("96.0.1", "Firefox 9601"),
```

Comment:
Should this be "96.0.1"?

This is what it currently looks like in prod for a draft experiment:
![Screen Shot 2022-01-11 at 8 56 26 AM](https://user-images.githubusercontent.com/557895/148987144-85a6b4a7-a1c0-4624-81a0-5d200ed7e658.png)

Reply:
Yeah you're not wrong but it's not setup right now to explicitly specify a nicer label, I'm sure @ncloudioj will know what it means.